qt/scientific_interfaces/ISISReflectometry/GUI/Save/QtSaveView.h

```diff
@@ -102,6 +102,13 @@ public slots:
   /// Initialize the interface
   void initLayout();
 
+  void onSettingsChanged();
+  void connectSettingsChange(QLineEdit &edit);
+  void connectSettingsChange(QComboBox &edit);
+  void connectSettingsChange(QCheckBox &edit);
+  void connectSettingsChange(QRadioButton &edit);
+  void connectInstrumentSettingsWidgets();
```

Comment:
Should this be named `connectSaveSettingsWidgets`?

Also, it does not appear to be called - I think this is just an omission, because the warnings aren't working at the moment.

Reply:
Yes, it should be, though I'm not sure where it should get called?